File: pmd/java/aulas/Calculadora.java
package aulas;

public class Calculadora {

  private double operando1, operando2;
  private double resultado;
  private char operacao;

  public Calculadora() {
    this.operando1 = 0;
    this.operando2 = 0;
    this.operacao = '+';
    this.resultado = 0;
  }

  public Calculadora(double operando1, double operando2, char operacao) {
    this.operando1 = operando1;
    this.operando2 = operando2;
    this.operacao = operacao;
    calcular();
  }

  public double getOperando1() {
    return operando1;
  }

  public void setOperando1(double operando1) {
    this.operando1 = operando1;
  }

  public double getOperando2() {
    return operando2;
  }

  public void setOperando2(double operando2) {
    this.operando2 = operando2;
  }

  public char getOperacao() {
    return operacao;
  }

  public void setOperacao(char operacao) {
    this.operacao = operacao;
  }

  public double getResultado() {
    return resultado;
  }

  public void setResultado(double resultado) {
    this.resultado = resultado;
  }

  public String toString() {
    return String.format("%.2f %c %.2f = %.2f",
        operando1, operacao, operando2, resultado);
  }

  public void calcular() {
    if (operacao == '+')
      resultado = operando1 + operando2;
    else if (operacao == '-')
      resultado = operando1 - operando2;
    else if (operacao == '*')
      resultado = operando1 * operando2;
    else if (operacao == '/')
      resultado = operando1 / operando2;
  }

}

File: pmd/python/aulas/calculadora.py
class Calculadora:

  def __init__(self, operando1=0.0, operando2=0.0, operacao="+"):
    self.operando1 = operando1
    self.operando2 = operando2
    self.operacao = operacao
    self.calcular()

  def __str__(self):
    return f"{self.operando1} {self.operacao} {self.operando2} = {self.resultado}"
    #return "%.2f %c %.2f = %.2f" % (self.operando1, self.operacao, self.operando2, self.resultado)

  def calcular(self):
    if self.operacao == "+": 
      self.resultado = self.operando1 + self.operando2
    elif self.operacao == "-": 
      self.resultado = self.operando1 - self.operando2
    elif self.operacao == "*": 
      self.resultado = self.operando1 * self.operando2
    elif self.operacao == "/": 
      self.resultado = self.operando1 / self.operando2



c = Calculadora()
opcao = None
while True:

  opcao = int(input("\n\n0 SAIR\n1 +\n2 -\n3 *\n4 \\\nOpcao:"))

  match opcao:
    case 0:
      exit(0)

    case 1:
      c.operando1 = float(input("Digite o 1o operando: "))
      c.operando2 = float(input("Digite o 2o operando: "))
      c.operacao = "+"
      c.calcular()
      print(c)
      
    